prep-javascript-conditionals/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Conditionals</title>
+</head>
+
+<body>
+  <script>
+    var age = 19;
+
+    var student1Score = 50;
+
+    var student2Score = 40;
+
+    var season = 'winter';
+
+    var dayOfTheWeek = 'wednesday';
+
+    if (age < 18) {
+      console.log('A child');
+    } else {
+      console.log('Adult');
+    }
+
+    if (student1Score >= 70) {
+      console.log('Congrats you received a passing grade!');
+    } else {
+      console.log('Oh no you failed!');
+    }
+
+    if (student2Score < 60) {
+      console.log('F');
+    } if (student2Score < 70) {
+      console.log('D');
+    } if (student2Score < 80) {
+      console.log('C');
+    } if (student2Score < 90) {
+      console.log('B');
+    } if (student2Score < 100) {
```

Comment:
for these middle `if` statements you want `else if` so that only the matching grade gets logged to the console